ckan/config/deploy.rb

```diff
@@ -23,6 +23,18 @@ def run_paster_command(command)
   task :install_package, roles: :app do
     run "cd #{release_path} && '#{virtualenv_path}/bin/python' #{release_path}/setup.py install"
   end
+
+  desc "Restart harvest gather process"
+  task :restart_harvest_gather_process do
+    run "sudo initctl restart harvester_gather_consumer-procfile-worker || sudo initctl start harvester_gather_consumer-procfile-worker"
+  end
+
+  desc "Restart harvest fetch process"
+  task :restart_harvest_fetch_process do
+    run "sudo initctl restart harvester_fetch_consumer-procfile-worker || sudo initctl start harvester_fetch_consumer-procfile-worker"
+  end
 end
 
 after "deploy:create_symlink", "deploy:install_package"
+after "deploy:install_package", "deploy:restart_harvest_gather_process"
```

Comment:
I think these should be `after "deploy:restart"` because then it will restart the workers even if we just request a restart without doing a deploy.

Reply:
ok, will update